dll/win32/iphlpapi/ipstats_reactos.c

```diff
@@ -150,8 +150,8 @@ NTSTATUS tdiGetRoutesForIpEntity
 ( HANDLE tcpFile, TDIEntityID *ent, IPRouteEntry **routes, PDWORD numRoutes ) {
     NTSTATUS status = STATUS_SUCCESS;
 
-    TRACE("TdiGetRoutesForIpEntity(tcpFile %x,entityId %x)\n",
-           (DWORD)tcpFile, ent->tei_instance);
+    TRACE("TdiGetRoutesForIpEntity(tcpFile 0x%lx, entityId 0x%x)\n",
+          (DWORD)tcpFile, ent->tei_instance);
```

Comment:
please use %p with tcpfile or %Ix (uppercase i!) and (DWORD_PTR)tcpfile.

Reply:
`0x%p`, 3 times.